matching-notification-api/src/main/resources/ApplicationResources.properties

```diff
@@ -181,3 +180,8 @@ phenotips.myMatches.contact.preview.error.header=Error getting contact email pre
 phenotips.myMatches.notification.dialog.success.title=Success
 phenotips.myMatches.notification.dialog.error.title=Error
 phenotips.myMatches.notification.dialog.closeButton.name=Close
+
+phenotips.myMatches.disclaimer.matchesLastRun=Matches last ran on
+phenotips.myMatches.disclaimer.noMatchRequest=Matches were never ran
```

Comment:
Now that I see it in the UI I'm not sure the texts I proposed in the Jira tasks are clear enough or even correct English.

My new suggestion would be "Matches with this database were last computed on <date>." and "Matches with this database were never computed".